utilities/backupable/backupable_db_test.cc

```diff
@@ -1019,6 +1019,31 @@ TEST_F(BackupableDBTest, BackupOptions) {
   CloseDBAndBackupEngine();
 }
 
+TEST_F(BackupableDBTest, SetOptionsBackupRaceCondition) {
+  OpenDBAndBackupEngine(true);
+  SyncPoint::GetInstance()->LoadDependency(
+      {{"CheckpointImpl::CreateCheckpoint:SavedLiveFiles1",
+        "BackupableDBTest::SetOptionsBackupRaceCondition:AfterSetOptions"},
```

Comment:
nit: if you want to make sure the race condition is exposed I think this should be a "BeforeSetOptions" sync point. Otherwise all the `SetOptions` could (theoretically) finish before the backup starts.

Reply:
Done.